src/crimson/os/seastore/journal.cc

```diff
@@ -339,77 +339,175 @@ std::optional<std::vector<delta_info_t>> Journal::try_decode_deltas(
   return deltas;
 }
 
+std::optional<std::vector<extent_info_t>> Journal::try_decode_extent_infos(
```

Comment:
nit, in the commit message. s/to forms of scan/two forms of scan/. it took me at three passes to convince myself it was a typo.

Reply:
Ok